### PR TITLE
Mark Courses as Complete or Incomplete from Student Profile

### LINKED ISSUE #31: Home Page: Add more details about athena
The home page is a little empty. We should add some bullet points about what athena is and what it does.

### PR DESCRIPTION
**Pull request by PeterETschaekofske**
issue resolved. 
TODO: pagination (#69)

Fixes #31

### CODE REVIEW

File: src/Athena.Data/Repositories/CourseRepository.cs [hunk 1]
```diff
@@ -129,6 +129,50 @@ public CourseRepository(IDbConnection db) : base(db)
...
+                       c.name,
+                       i.id,
+                       i.name,
+                       i.description
+                FROM courses c
+                    LEFT JOIN institutions i
+                        ON c.institution = i.id
+                WHERE c.id NOT IN (SELECT course FROM student_x_completed_course WHERE student = @student)
+                      AND c.name ILIKE ('%' || @q || '%')
+                      AND i.id IN (SELECT institution FROM institution_x_student WHERE student = @student)",
```
Comment (nlowe) [inline src/Athena.Data/Repositories/CourseRepository.cs]: Do we want to filter by enrolled programs as well?

File: src/Athena/Athena.csproj [hunk 2]
```diff
@@ -16,7 +16,6 @@
     <DotNetCliToolReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Tools" Version="2.0.2" />
   </ItemGroup>
   <ItemGroup>
-    <Folder Include="Content" />
```
Comment (nlowe) [inline src/Athena/Athena.csproj]: I **think** we need to keep this if you want the folder to show up in solution explorer but I'm not certain, we should double check this.

File: src/Athena/Content/js/ConfigureCompletedCourses.js [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+            setIncompleteCourses(data);
+        })
+        .fail(function (err) {
+            console.error("broke: ", err)
+        });
+}
+
+export function init(studentId) {
+    this.StudentId = studentId;
+    console.log("Init for student: ", studentId);
```
Comment (nlowe) [inline src/Athena/Content/js/ConfigureCompletedCourses.js]: Let's remove this logging statement, it's not super helpful.

File: src/Athena/Athena.csproj (hunk 2 above)
Comment (PeterETschaekofske) [reply src/Athena/Athena.csproj]: the folder is still there after closing and re-opening VS

File: src/Athena/Content/js/ConfigureCompletedCourses.js (hunk 3 above)
Comment (PeterETschaekofske) [reply src/Athena/Content/js/ConfigureCompletedCourses.js]: it shall be done

**Pull request merged**